### PR TITLE
Refactor to use our tamagui wrapper instead of native-base

### PR DESCRIPTION
**Pull request by mbonete**
Related to task:
https://github.com/users/andrew-bierman/projects/3/views/1?filterQuery=&pane=issue&itemId=34309618

### CODE REVIEW

File: client/screens/feed/Feed.tsx [hunk 1]
```diff
@@ -74,72 +69,70 @@ const FeedSearchFilter = ({
...
+              checked={selectedTypes.trip}
+              onCheckedChange={handleToggleTrip}
             />
-          </HStack>
+          </RStack>
         )}
-        <HStack space={3} alignItems="center">
-          <Text
-            fontSize="lg"
-            fontWeight="bold"
```
Comment (andrew-bierman) [inline client/screens/feed/Feed.tsx:138]: we might want bold still

Comment (mbonete) [reply client/screens/feed/Feed.tsx:138]: Thanks for the piece of feedback! I made that call while refactoring the design as I found the non-bold version closer to the apperance of the original view. I will put back the `bold` and prioritize keeping the HTML/design semantic over visual fidelity.

Comment (andrew-bierman) [reply client/screens/feed/Feed.tsx:138]: I like your thinking! that's a good thing to be aware of

**Comment by andrew-bierman** [comment]
Possible memory leak in the RStacks in FeedCard? Trying searching with search bar and see, super laggy for me and crashes. I narrowed it down to in here maybe ? But could use second opinion

**Comment by andrew-bierman** [comment]
Dropping a comment as a reminder about memory leak / performance issue when using FeedCard. I think it's the 2 nested Stacks inside the card for some reason, but not 100%. Does not happen with native base weirdly.

**Comment by andrew-bierman** [comment]
https://github.com/andrew-bierman/PackRat/issues/521

**Pull request merged**